### PR TITLE
Food gathering exp

### PR DESCRIPTION
**Pull request by laserprec**
Some changes to start training agents on food gathering

1. Expand arenas to 9 from 4 copies
2. Correct team ID so the two teams are adversarial (previously both have ID = 0)
3. Add config for POCA algorithm
4. Add training instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,20 @@ See more details in the Docs directory
...
+1. Open `ScavengerWorldUnity` project in Unity and in the editor select `Assets -> Scenes -> Training -> GathererTraining`
+1. Open up a terminal of your choice (and preferably a python virtual environment)
+1. `pip3 install torch~=1.7.1 -f https://download.pytorch.org/whl/torch_stable.html`
+1. `python -m pip install mlagents==0.28.0`
+1. cd to where you had `ScavengerWorldUnity` stored on disk
+1. To start training, run `mlagents-learn.exe configs\TrainingZones\FoodCollectionPoca.yml --run-id=FoodGatherExp1`
+1. Go back to the Unity Editor and hit the `play` button in the middle of the top screen.
+1. To monitor training metrics, on another terminal window and run `tensorboard --logdir results`
+1. Open the address `localhost:6006` in your browser to see the tensorboard dashboard.
+
```
Comment (logar16) [inline README.md:29]: Add a comment that you can build the game and then run it with `--no-graphics` for faster training.  

Also, I found that using a `--time-scale` of <20 (I think 5 was my best when using the editor) seems to keep it from getting jerky and slowing down.

**Review by logar16 — APPROVED** [review]
Looks great, only have one tiny addition to the instructions.  Thanks for adding those!

**Review by T00L00 — COMMENTED** [review]
The teams are automatically assigned upon starting the simulation, so we won't have to worry about manually setting them :)

File: README.md (hunk 1 above)
Comment (laserprec) [reply README.md:29]: There's some build errors when we package the game. We can address that and this instruction in a separated PR.

**Pull request merged**